peer_heap.js

```diff
@@ -44,25 +46,6 @@ function PeerHeap(random) {
     this.maxRangeStart = 0;
 }
 
-PeerHeap.prototype.choose1 = function choose1(threshold, filter) {
```

Comment:
i removed the micro op because the logic got messy.

I think optimizing for 1 or 2 peers isn't very common in prod.